src/UI/class-settings-page.php

```diff
@@ -218,62 +219,45 @@ private function initialize_basic_section(): void {
 		$field_id   = 'apikey';
 		$field_args = array(
 			'option_key'    => $field_id,
-			'help_text'     => __( 'Your Site ID is typically your own site domain without <code>http(s)://</code> prefixes or trailing <code>/</code> (e.g. <code>mydomain.com</code>).', 'wp-parsely' ),
+			'help_text'     => $are_credentials_managed ? '' : __( 'Your Site ID is typically your own site domain without <code>http(s)://</code> prefixes or trailing <code>/</code> (e.g. <code>mydomain.com</code>).', 'wp-parsely' ),
```

Comment:
Do you think it's better to have the help text? I removed it since users won't be interacting with the field, thus I deemed it has no practical value.

Reply:
OK, done.